pinot-clients/pinot-jdbc-client/src/main/java/org/apache/pinot/client/PinotConnection.java

```diff
@@ -38,11 +40,16 @@
 public class PinotConnection extends AbstractBaseConnection {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(Connection.class);
+  protected static final String[] POSSIBLE_QUERY_OPTIONS = {
+    QueryOptionKey.ENABLE_NULL_HANDLING,
+    QueryOptionKey.USE_MULTISTAGE_ENGINE
+  };
   private org.apache.pinot.client.Connection _session;
   private boolean _closed;
   private String _controllerURL;
   private PinotControllerTransport _controllerTransport;
-  private final boolean _enableNullHandling;
+  private final HashMap<String, Boolean> _queryOptions = new HashMap<String, Boolean>();
```

Comment:
use `Map<String, String> _queryOptions` for future extensibility.

Reply:
WIll do.